### PR TITLE
Undoing breaking changes to the cve object, adding deprecation tags instead.

### PR DESCRIPTION
**Pull request by floydtree**
#### Related Issue: 
Great find by @Aniak5 during release note preparation. Original PR - https://github.com/ocsf/ocsf-schema/pull/678. This change was brought in before we had the concept of `@deprecated` tag.

#### Description of changes:
1. Adding back removed attributes with deprecation tag in the dictionary.
2. Undoing changes in the cve object

Screenshot 2024-01-23 at 14 32 12

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# CHANGELOG
+All notable changes to this project will be documented in this file. `[Unreleased]` section at the top, will be used to track upcoming changes.
```
Comment (Aniak5) [inline CHANGELOG.md:2]: Curious, did we want to add this in with this PR or in a differnet one?

Comment (floydtree) [reply CHANGELOG.md]: Nope, that slipped in, removed that file.

**Review by Aniak5 — APPROVED** [review]
LGTM! Thanks for fixing this @floydtree

**Review by mikeradka — APPROVED** [review]
Awesome catch @Aniak5 and @floydtree got this in so quickly 🍺

**Review by pagbabian-splunk — APPROVED** [review]

**Pull request merged**